deployment/GP03FLMA_Deploy.csv

```diff
@@ -107,39 +107,39 @@ SR1811,,SKQ201920S,,GP03FLMA-RIS01-00-SIOENG000,6,1,2018-07-23T22:54:00,2019-09-
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-ADCPSL003,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-ADCPSL-23497,50.0227,-144.360783,,500,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOG040,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOG-10216,50.0227,-144.360783,,30,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOG041,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOG-11609,50.0227,-144.360783,,40,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOG042,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOG-11637,50.0227,-144.360783,,60,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOG043,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOG-11646,50.0227,-144.360783,,90,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOG044,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOG-12587,50.0227,-144.360783,,130,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOG045,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOG-12309,50.0227,-144.360783,,180,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOG046,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOG-11593,50.0227,-144.360783,,250,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOG047,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOG-11535,50.0227,-144.360783,,350,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOG048,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOG-12308,50.0227,-144.360783,,500,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOH049,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOH-11683,50.0227,-144.360783,,750,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOH050,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOH-10264,50.0227,-144.360783,,1000,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOH051,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOH-12389,50.0227,-144.360783,,1500,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIS01-00-SIOENG000,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGCON-SIOCON-14876,50.0227,-144.360783,,30,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIS01-03-DOSTAD000,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-DOSTAD-00379,50.0227,-144.360783,,30,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIS01-04-PHSENF000,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-PHSENF-P0103,50.0227,-144.360783,,30,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIS01-05-FLORTD000,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-FLORTD-01122,50.0227,-144.360783,,30,4103,,,
```

Comment:
Recovery times look good.  All times are anchor release; mid-water release failed to fire.

Reply:
Looks good.